src/t_string.c

```diff
@@ -77,6 +77,9 @@ static int getExpireMillisecondsOrReply(client *c, robj *expire, int flags, int
 
 void setGenericCommand(client *c, int flags, robj *key, robj *val, robj *expire, int unit, robj *ok_reply, robj *abort_reply) {
     long long milliseconds = 0; /* initialized to avoid any harmness warning */
+    int keyFound = false;
+    int keyLooked = false;
```

Comment:
Redis doesn't use true/false, it's not defined for some of the platforms we build against. 
```suggestion
    int keyFound = 0;
    int keyLooked = 0;
```